content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -59,6 +59,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
 * The [ngrok Kubernetes Ingress Controller](https://github.com/ngrok/kubernetes-ingress-controller) is an open source controller for adding secure public access to your K8s services using the [ngrok platform](https://ngrok.com).
 * The [OCI Native Ingress Controller](https://github.com/oracle/oci-native-ingress-controller#readme) is an Ingress controller for Oracle Cloud Infrastructure which allows you to manage the [OCI Load Balancer](https://docs.oracle.com/en-us/iaas/Content/Balance/home.htm).
+* [OpenNJet Ingress Controller](https://gitee.com/njet-rd/open-njet-kic) is a [OpenNJet](https://njet.org.cn/)-based ingress controller.
```

Comment:
It's not suitable to add a project hosted on [gitee.com](https://gitee.com)

Reply:
I don't know of any reason for concern _by Kubernetes_. Although there have been complaints of censorship, we allow links to closed-source software as well as truly open source projects.

Reply:
Can I link to gitee?

Reply:
You can.

Reply:
> You can.

What else is wrong with my PR?Is it acceptable?